pydarn/utils/coordinates.py

```diff
@@ -20,15 +20,17 @@
 """
 import enum
 
-class Coord(enum.Enum):
+class Coords(enum.Enum):
```

Comment:
Aaaaa does this break the ACF plotting code? be careful replacing already established method names. This can break code.  I would almost suggest reverting for safety's sake if others agree.

Reply:
I had a circle loop error when I used just Coord to call the coord type. However, that stopped when I changed the name to Coords. I was also thinking since we will not just be using one Coordinate it will be reasonable to name it Coords to implying not just one coordinate system.